setup.py

```diff
@@ -33,6 +36,23 @@ def parse_requirements_file(fname):
     raise RuntimeError('Could not determine version')
 
 
+class _FillDocsPyCommand(build_py):
+    def run(self):
+        # Currently a noop, but shows where we can rewrite (somehow)
+        super().run()
+        target_dir = Path(self.build_lib)
+        assert target_dir.is_dir()
+        for root, dirs, files in os.walk(target_dir, topdown=False):
+            for fname in files:
+                if fname.endswith('.py'):
+                    fname = op.join(root, fname)
+                    with codecs.open(fname, 'r', 'utf8') as fid:
+                        contents = fid.readlines()
+                    with codecs.open(fname, 'w', 'utf8') as fid:
+                        for line in contents:
+                            fid.write(line)
```

Comment:
@hoechenberger this is where the rewriting would happen, it gets called in `setup.py` in this PR. If you want, you could try modifying this to write even just one hard-coded `.pyi` file to see if vscode picks it up properly.